build.gradle

```diff
@@ -21,7 +21,7 @@ dependencies {
 	modCompile "net.fabricmc:fabric-loader:${project.loader_version}"
 
 	// Fabric API. This is technically optional, but you probably want it anyway.
-	modCompileOnly "net.fabricmc.fabric-api:fabric-api:${project.fabric_version}"
+	modCompileOnly "net.fabricmc.fabric-api:fabric-rendering-v1:${project.fabric_rendering_v1}"
```

Comment:
With this I was referring to something more like:

`fabricApi.module("fabric-rendering-v1", project.fabric_version)`

Since that would allow using a global FAPI version in case at some point hooks with other modules were implemented, so there's no need to track individual versions for each module.